src/main/scala/cromwell/engine/CallExecutionActor.scala

```diff
@@ -70,6 +71,9 @@ class CallExecutionActor(backendCall: BackendCall) extends Actor with CromwellAc
   def withRetry(work: Future[ExecutionHandle], onSuccess: ExecutionHandle => Unit, onFailure: => Unit): Unit = {
     work onComplete {
       case Success(s) => onSuccess(s)
+      case Failure(e: GoogleJsonResponseException) if e.getStatusCode == 403 =>
+        logger.error(e.getMessage, e)
+        throw e
```

Comment:
This is only called from the vassal receive and not wrapped with a `Try` or `Failure`, so it looks like this will crash the vassal.  There's currently no supervision of vassals that would restart it.

Assuming we keep essentially the current structure, I think `withRetry` will need to make a proper distinction between events that should be retried and those that should not.